.cirrus.yml

```diff
@@ -30,6 +31,8 @@ low_scale_task:
     - tar -xzf runtime-cache/runtime.tar.gz || true
 
   install_script:
+    - export IP_ADDR=$(ip route get 8.8.8.8 | head -1 | sed 's/.*src \([0-9\.]*\).*/\1/')
+    - 'sed -i "s/<ip>/${IP_ADDR}/g" ${PHYS_DEPLOYMENT}'
```

Comment:
You also need to change the `configure_ssh_script` step, because it only configures SSH for `localhost`.